api-reference/10 UI Widgets/dxTreeView/1 Configuration/createChildren.md

```diff
@@ -81,6 +81,69 @@ The following code shows how to use this function with a remote service:
         // ...
     })
 
+##### Vue
+
+    <!-- tab: App.vue -->
+    <template>
+        <div>
+            <DxTreeView
+            :create-children="createChildren"
+            :root-value="''"
+            data-structure="plain"
+            />
+        </div>
+    </template>
+    <script>
+
+    import DxTreeView from 'devextreme-vue/tree-view';
+    import 'whatwg-fetch';
+
+    export default {
+        components: {
+            DxTreeView
+        },
+        methods: {
+            createChildren: function(parent) {
+            let parentId = parent ? parent.itemData.id : '';
+
+            return fetch(`https://js.devexpress.com/Demos/Mvc/api/TreeViewData?parentId=${parentId}`)
+                .then(response => response.json())
+                .catch(() => { throw 'Data Loading Error'; });
+            }
+        }
+    };
+    </script>
+
+##### React
+
+    <!-- tab: App.js -->
+    import React from 'react';
+    import TreeView from 'devextreme-react/tree-view';
+    import 'whatwg-fetch';
+
+    class App extends React.Component {
```

Comment:
We need to use hooks in React. You can use my suggestion in Eugeniy's PR: https://github.com/DevExpress/devextreme-documentation/pull/1554/files#r539252162

Reply:
Great idea.
Fixed - https://github.com/DevExpress/devextreme-documentation/pull/1555/commits/7b6f4fd20f624f56d56ed23aa3636d1f255bd9ff